db/migrate/20190723092536_fix_fe_course_requirements.rb

```diff
@@ -0,0 +1,9 @@
+class FixFeCourseRequirements < ActiveRecord::Migration[5.2]
+  def up
+    say_with_time 'setting further education course requirements to not_required' do
```

Comment:
I thought this was a `say/espeak` invocation for a second...

Reply:
That would be awesome.